app/validators/idv/form_state_id_validator.rb

```diff
@@ -11,14 +11,24 @@ module FormStateIdValidator
                 presence: true
 
       validates_with UspsInPersonProofing::TransliterableValidator,
-                     fields: [:first_name, :last_name],
+                     fields: [:first_name, :last_name, :state_id_city],
                      reject_chars: /[^A-Za-z\-' ]/,
                      message: ->(invalid_chars) do
                        I18n.t(
                          'in_person_proofing.form.state_id.errors.unsupported_chars',
                          char_list: invalid_chars.join(', '),
                        )
                      end
+
+      validates_with UspsInPersonProofing::TransliterableValidator,
+                     fields: [:state_id_address1, :state_id_address2],
+                     reject_chars: /[^A-Za-z0-9\-' .\/#]/,
+                     message: ->(invalid_chars) do
+                       I18n.t(
+                         'in_person_proofing.form.state_id.errors.unsupported_chars',
```

Comment:
Messaging needs to be different here.

```suggestion
                         'in_person_proofing.form.address.errors.unsupported_chars',
```